nexus/tests/output/nexus_tags.txt

```diff
@@ -22,48 +24,48 @@ spoof_login                              /login
 
 API operations found with tag "images"
 OPERATION ID                             URL PATH
-project_images_delete_image              /organizations/{organization_name}/projects/{project_name}/images/{image_name}
-project_images_get                       /organizations/{organization_name}/projects/{project_name}/images
-project_images_get_image                 /organizations/{organization_name}/projects/{project_name}/images/{image_name}
-project_images_post                      /organizations/{organization_name}/projects/{project_name}/images
+image_create                             /organizations/{organization_name}/projects/{project_name}/images
+image_delete                             /organizations/{organization_name}/projects/{project_name}/images/{image_name}
+image_list                               /organizations/{organization_name}/projects/{project_name}/images
+image_view                               /organizations/{organization_name}/projects/{project_name}/images/{image_name}
 
 API operations found with tag "images:global"
```

Comment:
Maybe unrelated to this PR, but why is the tag "images:global" and not "global_images"?

Reply:
yeah; let's set that aside? I agree though that it's ... odd

Reply:
sounds good!

Reply:
Maybe we should have a single tag for fleet-wide operations?

Reply:
That sounds really good!